Server/Plugins/APIDump/main_APIDump.lua

```diff
@@ -1635,6 +1635,73 @@ end
 
 
 
+local function DumpLuaCheck(a_API)
+	LOG("Creating file .luacheckrc...")
+	local file = io.open(".luacheckrc", "w")
+	
+	local Content = [[
+-- Ignore unused function and loop arguments
+unused_args = false
+
+-- Allow self defined globals
+allow_defined = true
+
+-- Ignore this functions
+ignore =
+{
+	"Initialize",  -- Plugin
+	"OnDisable",  -- Plugin
+	"RegisterPluginInfoCommands",  -- InfoReg.lua
+	"RegisterPluginInfoConsoleCommands",  -- InfoReg.lua
+	"g_PluginInfo",  -- Info.lua
+}
+
+-- Ignore files / directories
+exclude_files =
+{
+	"tests/" -- CuberitePluginChecker
+}
+
+-- All globals from cuberite (classes, enums, functions)
+globals =
+{
+]]
+
+	file:write(Content)
```

Comment:
You could write the text to the file directly, without assigning it to `Contents`.